tests/test_torch_dftd3_calculator_benchmark.py

```diff
@@ -4,19 +4,30 @@
 import numpy as np
 import pytest
 from ase import Atoms
-from ase.build import fcc111, molecule
+from ase.build import bulk, fcc111, molecule
 from ase.calculators.dftd3 import DFTD3
 from ase.units import Bohr
 from torch_dftd.torch_dftd3_calculator import TorchDFTD3Calculator
 
 
-def _create_atoms() -> List[Atoms]:
+@pytest.fixture(
+    params=[
+        pytest.param("mol", id="mol"),
+        pytest.param("slab", id="slab"),
+        pytest.param("large", marks=[pytest.mark.slow], id="large"),
+    ]
+)
+def atoms(request) -> Atoms:
```

Comment:
I added larger system test with `pytest.mark.slow` custom marker (won't run on CI because CI has `-m "not slow"` option). `atoms` is defined as `pytest.fixture` now.